modules/oadp-restic-timeouts.adoc

```diff
@@ -8,7 +8,7 @@
 
 `timeout` defines the Restic timeout. The default value is `1h`.
 
-Use the Restic `timeout` for the following scenarios:
```

Comment:
I would go with the term "parameter" or similar. As in:

Use the Restic `timeout` parameter in the `nodeAgent` section for the following scenarios:

I would change the previous paragraph too to say:

The `spec.configuration.nodeAgent.timeout` parameter defines the Restic timeout. The default value is `1h`.

You don't have to go with these exact formulas, but one generally does need to state the kind of object a name denotes, not just the name.

Reply:
Done. Thanks.